### PR TITLE
WIP: CI Workflows

### PR DESCRIPTION
**Pull request by xermicus**
The seemingly random code changes are to make clippy happy; I thought I just drive by them as its only a hand full

### CODE REVIEW

**Review by ascjones — COMMENTED** [review]
I believe it is missing installing the `riscv` toolchain

**Comment by xermicus** [comment]
Most definitively

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          rustup override set stable
+          rustup update stable
+      - name: Install RV64E toolchain
+        run: |
+          wget https://github.com/paritytech/rustc-rv32e-toolchain/releases/download/v1.1.0/rust-rve-nightly-2024-01-05-x86_64-unknown-linux-gnu.tar.zst -O rust-rve-nightly.tar.zst
+          tar --zstd -xf rust-rve-nightly.tar.zst
+          mv rve-nightly ~/.rustup/toolchains/
+          rm rust-rve-nightly.tar.zst
+      - name: Install cargo contract
+        run: cargo install --git https://github.com/paritytech/cargo-contract.git --branch at/riscv --force --locked
```
Comment (ascjones) [inline .github/workflows/pr.yml]: I think this is unnecessary...the `ink` build calls directly into the `contract-build` crate

Comment (ascjones) [reply .github/workflows/pr.yml]: https://github.com/ascjones/schlau/blob/8920b85b2111f769c52be5877f18ec1239929546/src/ink.rs#L164

**Review by ascjones — APPROVED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (xermicus) [reply .github/workflows/pr.yml]: True that, and other than that it's probably not required for now

**Pull request merged**